### ISSUE TITLE
Fix warnings

### ISSUE THREAD

**Issue by mperham** [celluloid/celluloid#402]
```
/Users/mikep/.gem/ruby/2.0.0/gems/celluloid-0.15.2/lib/celluloid/call_chain.rb:13: warning: mismatched indentations at 'end' with 'module' at 1
/Users/mikep/.gem/ruby/2.0.0/gems/celluloid-0.15.2/lib/celluloid/tasks.rb:76: warning: global variable `$CELLULOID_DEBUG' not initialized
```

**Comment by JuanitoFatas** [comment]
Celluloid 0.16.0

```
D, [2015-03-18T02:26:51.673251 #25738] DEBUG -- : Terminating 3 actors...
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/tasks.rb:79: warning: global variable `$CELLULOID_DEBUG' not initialized
W, [2015-03-18T02:26:51.673602 #25738]  WARN -- : Terminating task: type=:finalizer, meta={:method_name=>:__shutdown__}, status=:receiving
	Celluloid::TaskFiber backtrace unavailable. Please try `Celluloid.task_class = Celluloid::TaskThread` if you need backtraces here.
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/tasks.rb:79: warning: global variable `$CELLULOID_DEBUG' not initialized
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/mailbox.rb:149: warning: global variable `$CELLULOID_DEBUG' not initialized
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/mailbox.rb:149: warning: global variable `$CELLULOID_DEBUG' not initialized
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/mailbox.rb:149: warning: global variable `$CELLULOID_DEBUG' not initialized
/Users/Juan/.gem/ruby/2.2.1/gems/celluloid-0.16.0/lib/celluloid/mailbox.rb:149: warning: global variable `$CELLULOID_DEBUG' not initialized
```

**Comment by tarcieri** [comment]
Yeah definitely, this is unfortunately some contributed code that should've probably been more thoroughly audited.

**Comment by digitalextremist** [comment]
I believe this was resolved as of #518? There are new warnings, but the ones mentioned here are gone.

**Comment by tarcieri** [comment]
Yeah, this is fixed now.
